e2e-tests/insurance/cypress/e2e/journeys/complete-eligibility-answers-for-a-second-time.spec.js

```diff
@@ -70,8 +70,9 @@ context('Insurance - Eligibility - start and complete for a second time after cr
     completeAndSubmitBuyerCountryForm();
 
     // insured amount question
-    cy.assertUncheckedYesNoRadios();
-    cy.completeInsuredAmountForm();
+    cy.assertTotalValueInsuredRadios({ checked: false });
+    cy.assertTotalValueInsuredRadios({ secondOption: true, checked: false });
```

Comment:
Why is it being asserted twice here? Both radios should be unchecked here

Could we rename this as well?
```
// insured amount question
```

Reply:
so it by default goes to firstOption as true so checks both.  Added secondOption as false to be clearer

Reply:
Why is the first option true? So far, when starting eligibility from scratch (or for a second time), no answers should be pre-populated